pkgs/applications/radio/noaa-apt/default.nix

```
@@ -0,0 +1,69 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, atk
+, cairo
+, gdk-pixbuf
+, glib
+, gtk3
+, openssl
+, pango
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "noaa-apt";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "martinber";
+    repo = "noaa-apt";
+    rev = "v${version}";
+    sha256 = "0fqki4a9c54rixdz5bpswvn433f9saw6yazgw4av3xdd7g2fdvvj";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    atk
+    cairo
+    gdk-pixbuf
+    glib
+    gtk3
+    openssl
+    pango
+  ];
+
+  enableParallelBuilding = true;
+
+  cargoSha256 = "1pzcq31inazmc9cz31fspzqkp0lpkjid8ai3g17sin1pfzby5jlh";
+
+  preBuild = ''
+    # Used by macro pointing to resource location at compile time.
+    export NOAA_APT_RES_DIR=$out/share/noaa-apt
+  '';
+
+  postInstall = ''
+    # Resources.
+    mkdir -p $out/share/noaa-apt
+    cp -R $src/res/* $out/share/noaa-apt/
+
+    # Desktop icon.
+    mkdir -p $out/share/applications
+    mkdir -p $out/share/icons/hicolor/48x48/apps
+    mkdir -p $out/share/icons/hicolor/scalable/apps
+    cp $src/debian/ar.com.mbernardi.noaa-apt.desktop $out/share/applications/
+    cp $src/debian/noaa-apt.png $out/share/icons/hicolor/48x48/apps/
+    cp $src/debian/noaa-apt.svg $out/share/icons/hicolor/scalable/apps/
+  '';
+
+  meta = with lib; {
+    description = "NOAA APT image decoder.";
+    homepage = "http://noaa-apt.mbernardi.com.ar/";
+    license = licenses.gpl3;
```

`gpl3` is deprecated, use either `gpl3Only` or `gpl3Plus`. Can't find the 'or later wording anywhere, so we should probably assume `gpl3Only`:

```suggestion
    license = licenses.gpl3Only;
```

---

Done.